modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/datapage/DataPageReader.java

```diff
@@ -54,129 +49,49 @@ public DataPageReader(PageMemory pageMemory, int groupId, IoStatisticsHolder sta
     }
 
     /**
-     * Returns a row by link. To get the row bytes, more than one pages may be traversed (if the corresponding row
-     * was fragmented when stored).
+     * Traverses page memory starting at the given link. At each step, reads the current data row and feeds it to the given
+     * {@link PageMemoryTraversal} object which updates itself (usually) and returns next link to continue traversal
+     * (or {@link PageMemoryTraversal#STOP_TRAVERSAL} to stop.
      *
      * @param link Row link
-     * @return row object assembled from the row bytes
+     * @param traversal object consuming payloads and controlling the traversal
```

Comment:
Params descriptions usually start with a capital letters. Please try to adapt your code style to what you see in the project, not what technically counts as appropriate according to TC. We should all try to write code in similar manner, it's very important

Reply:
Fixed